components/ai_chat/resources/page/api/index.ts

```diff
@@ -3,23 +3,22 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at https://mozilla.org/MPL/2.0/. */
 
-import * as mojom from 'gen/brave/components/ai_chat/core/common/mojom/ai_chat.mojom.m.js'
-export * from 'gen/brave/components/ai_chat/core/common/mojom/ai_chat.mojom.m.js'
-import { debounce } from '$web-common/debounce'
-import { loadTimeData } from '$web-common/loadTimeData'
+ import { loadTimeData } from '$web-common/loadTimeData'
+ import API from '../../common/api'
+ import * as Mojom from '../../common/mojom'
```

Comment:
Because it's a module which defines many types and the consensus seems to be upper-case for module and types, and lower-case for class instances, functions, and variables, e.g.
```
import * as React from 'react'
```

what do you think?

Reply:
no strong feelings on it tbh, as long as we're consistent

FWIW, all our existing imports use the lowercase `m` :laughing:

![image](https://github.com/user-attachments/assets/bf1c6be9-f0d7-441c-82cf-b2bca08034b0)